### ISSUE TITLE
Fix phenology example

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-processes#72]
The Phenology example has invalid examples and is missing a description what the files are for.

Some problems I found in timeseries_udf.json: 
- NaN is used instead of null, but NaN is not valid JSON.
- There's a `crs` field in the GeoJSON of the polygons parameter of aggregate_polygon. That's misleading.
- Process get_collection is used instead of load_collection
- filter_bbox is used instead of filter_temporal
- filter_temporal uses from and to parameters instead of an array.

Some of the issues also exist in evi_cube.json.

To make the examples easier to read, I'd also remove the parameters which hold their default values (e.g. `"result": false` or `"crs": "EPSG:4326"`) or have floating point inconsistencies (e.g. `0.0006000000000000001`)

Having invalid examples in the master branch will confuse users and therefore we need to either fix it or revert the changes in the next days.

ToDo list:
- [ ] Add a README.md (link to it from the example overview page) and explain what the use case is and what the files in the folder are useful for.
- [ ] Fix the examples.

**Comment by jdries** [comment]
I started a pull request for this, but may still need to resolve some of the issues.
I updated the process graphs, as they were not updated to the latest versions that the notebook generated.
These graphs are in fact not needed by the notebook, but I was thinking that someone might find them useful as examples of advanced process graphs. Of course, this does require us to keep them up to date, so I'm fine with removing them if they cause issues.
